ui-testing-core/src/main/java/com/avito/android/test/UITestConfig.kt

```
@@ -41,4 +43,13 @@ object UITestConfig {
         AssertionError::class.java,
         PerformException::class.java
     )
+
+    /**
+     * Because of clicks implementation inside Espresso sometimes clicks can be interpreted
+     * as long clicks. Here we have several options to handle it.
+     */
+    sealed class ClickRollbackPolicy {
```

Let's add a fail fast implementation and use it as a default policy.